src/v2/Apps/Auction/Components/Form/initialValues.ts

```diff
@@ -1,34 +1,52 @@
+import { SavedAddressType } from "v2/Apps/Order/Utils/shippingUtils"
 import { Address } from "v2/Components/AddressForm"
+import { BillingInfoFormValues } from "./formValidation"
 
-export interface FormValuesForRegistration {
-  address?: Address
+export interface BillingInfoWithTerms {
+  address?: SavedAddressType | Address
   creditCard?: string
   agreeToTerms: boolean
 }
 
-export interface FormValuesForBidding extends FormValuesForRegistration {
+export interface BillingInfoWithBid extends BillingInfoWithTerms {
   selectedBid: string
 }
 
-const address: Address = {
+export interface BillingInfo {
+  address?: Address | SavedAddressType
+  isDefault?: boolean
+  internalID?: string
+}
+
+export const initAddress: Omit<Address, "phoneNumber"> | SavedAddressType = {
   name: "",
   addressLine1: "",
   addressLine2: "",
   country: "US",
   city: "",
   region: "",
   postalCode: "",
+  isDefault: false,
+}
+
+export const initAddressWithPhone: Address | SavedAddressType = {
+  ...initAddress,
   phoneNumber: "",
 }
 
-export const initialValuesForRegistration: FormValuesForRegistration = {
-  address,
+export const initBillingInfoWithTerms: Pick<
+  BillingInfoFormValues,
+  "address" | "agreeToTerms" | "creditCard"
```

Comment:
Although `addressWithPhone` exists as a key of the type `BillingInfoFormValues`, we never use the initial values as `addressWithPhone` but always as `address`. As far as I understand, we should never Pick `addressWithPhone`, right? I can be missing something, though.

Reply:
@lean8086 That's correct, because we'll never call addressWithPhone out of the formik context. It's always sent to us or back to the API under the umbrella of address. I think the intention here was to make it work, then (in a follow up PR or ticket) standardize the forms and their data sources to use 'phone number' as a key external to the address.

@araujobarret It is confusing! 😆 Most of these alternate cases stem from the data formatting inconsistencies we talked about. Hopefully in some follow up PRs we'll be able to standardize and clean up some of these different "initial values" objects.